### PR TITLE
fix: pass `batch_size` args to CE evaluators

### PR DESCRIPTION
**Pull request by omkar-334**
`batch_size` args are taken as input but not being passed to 2 of the CE evaluators. This PR fixes that.  

**Duplicate PRs that can be closed**
#2284 led me to this. It can probably be closed now?  
#1085, related to this, can be closed as well I think..
#1092, adds gradient accumulation for CE, i think this can be done thru `gradient_accumulation_steps` [Training Argument](https://sbert.net/docs/sentence_transformer/training_overview.html#training-arguments)

### CODE REVIEW

**Review by tomaarsen — APPROVED** [review]
This is great, thank you! I'll also have a look at the issues/PRs you linked.

- Tom Aarsen
